github-management/README.md

```diff
@@ -0,0 +1,65 @@
+# GitHub Management
+
+The Kubernetes project uses Github extensively to store and organize code,
+manage issues and documentation, and provide a consistent contributor flow.
+
+With the size and growth of the Kubernetes project, management of our Github
+footprint has historically been a challenge. We have created a number of
+policies to reduce friction and ease administration of our Github repositories
+and organizations. We have also created a number of tools to automate setup and
+enforcement of these policies.
+
+## Guides
+- [Organization Owners Guide](org-owners-guide.md)
+- [Repository Creation Guidelines](kubernetes-repositories.md)
+- [Setting up the CNCF CLA Check](setting-up-cla-check.md)
+- [GitHub Permissions](permissions.md)
+
+## Project Owned Organizations
+
+The following organizations are currently known to be part of the Kubernetes
+project
+
+### Actively used GitHub Organizations
+
+| Name | Description |
+| :--: | :---------: |
+| [kubernetes](https://github.com/kubernetes) | Core |
+| [kubernetes-client](https://github.com/kubernetes-client) | API Client Libraries |
+| [kubernetes-csi](https://github.com/kubernetes-csi) | Container Storage Interface Components |
+| [kubernetes-incubator](https://github.com/kubernetes-incubator) | Legacy Incubator Projects |
```

Comment:
Is this still active?  Don't we want to suggest kubernetes-sigs (subprojects) instead?  

https://github.com/kubernetes/community/blob/master/incubator.md

Reply:
It's still active in that there are still repos there, and the steering committee isn't pushing them to migrate anywhere right now. If it's in incubator, it can stay there for the time being, but no new projects will be added.